internal/infrastructure/db/fin_test.go

```diff
@@ -126,3 +126,54 @@ func TestUpdateActionPrivateData_failToUpdateBeforeStart(t *testing.T) {
 	err := repo.UpdateActionPrivateData(testUID, []byte("test-private-data"))
 	require.Error(t, err)
 }
+
+func TestGetBackupMetadata_success(t *testing.T) {
```

Comment:
The test for Get method is very similar to Set method. I think it's okay to unify these. This is not critical, so it is not required.

Reply:
I unified them: https://github.com/cybozu-go/fin/compare/27416f02c8e3c795d9bd7fcd2c428451dc940dfc..62adf0d5bb08541a979031ba93037443a22ea34b